### ISSUE TITLE
need to reconcile coordination pattern

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/upheno#619]
HPO uses a PATO class, MP uses an NBO class.

1. Which do we prefer to use?
2. If we go with the NBO class, is increased/decreased rate the right quality?

**HP_0011443 'Abnormality of coordination'**
'has part' some 
    (coordination
     and ('inheres in' some 'multicellular organism')
     and ('has modifier' some abnormal))

**MP_0001405 'impaired coordination'**
'has part' some 
    ('decreased rate'
     and ('inheres in' some 'motor coordination')
     and ('has modifier' some abnormal))

I drafted a pattern here, I'll revise it accordingly once we are in agreement:
https://github.com/obophenotype/upheno/pull/610

**Comment by sbello** [comment]
In this case I think I prefer the NBO term. The MP also makes use of child terms of the NBO term like limb coordination and gait. I'm not comfortable with PATO having increased and decreased coordination as child terms of coordinated. Why even insert the coordinated term?
I would say that rate is not the right quality to use. Maybe we could use increased/decreased process quality?

**Comment by nicolevasilevsky** [comment]
thanks @sbello!

**Comment by pnrobinson** [comment]
hmm I have not been a massive fan of NBO because it is not really the b building block approach of PATO for many things. Admittedly this is a hard area. But no really strong feelings

**Comment by matentzn** [comment]
I agree with @sbello on both accounts: 1) NBO should be favoured over PATO and 2) increased rate sounds wrong -> I also think increased process quality may work a bit better here, despite being comparatively devoid of meaning. 

I also agree Peter that NBO needs a lot of love, but for now, we have different groups contributing to NBO, and decided to go with it - but only as a source of what I would refer to as "atomic behaviours", i.e. behaviour terms like `smoking`, `walking`, `fleeing from enemy` etc. All the other things, in particular phenotypes, I encourage to be avoided.

**Comment by nicolevasilevsky** [comment]
Thanks @sbello and @matentzn 
I created PRs:
https://github.com/obophenotype/upheno/pull/622
https://github.com/obophenotype/upheno/pull/610
